### ISSUE TITLE
Fix color contrast issues of text on the Contracting page.

### ISSUE THREAD

**Issue by BlakeOrgan** [CivicActions/civicactions-homesite#261]
**Expected behavior**  
The heading titled "Our Expertise" should be the primary blue color #162E51

**Describe the bug**  
White text on a white background is causing a color contrast error where the text is not visible.

**To Reproduce**  
List the steps to reproduce the bug:

1.  Go to the [Contracting page](https://civicactions.com/contracting/) of CivicActions.com
2.  Scroll down to the section showing our service offerings
3.  Notice that the heading of this section is not visible unless you highlight it with your mouse.

**Image**

![](https://github.com/CivicActions/civicactions-homesite/assets/113871419/d9d1a86a-eef0-4f00-8a50-ee194b4b8d04)
